src/assets/currencies/aud.svg

```diff
@@ -1,22 +1,50 @@
+	l-6.6,13.6l11.9,9.4l-14.8,3.3l0.1,15.2l-11.8-9.5L376,168l0-15.2l-14.8-3.3l11.9-9.4L366.6,126.5z M338.9,250.8l-11.8-9.5
+	l-11.8,9.5l0-15.1l-14.8-3.3l11.9-9.4l-6.6-13.6l14.8,3.4l6.5-13.7l6.5,13.7l14.8-3.4l-6.6,13.6l11.9,9.4l-14.8,3.3L338.9,250.8z
+	 M399.7,395.7l-11.8-9.5l-11.8,9.5l0-15.1l-14.8-3.3l11.9-9.4l-6.6-13.6l14.8,3.4l6.5-13.7l6.5,13.7l14.8-3.4l-6.6,13.6l11.9,9.4
+	l-14.8,3.3L399.7,395.7z M416.4,292.2l-13.4-9.8l-13.5,9.8l5.1-15.8l-13.5-9.8h16.6l5.1-15.8l5.1,15.8h16.6l-13.5,9.8L416.4,292.2z
+	 M452.7,230.1l-11.8-9.5l-11.8,9.5l0-15.1l-14.8-3.3l11.9-9.4l-6.6-13.6l14.8,3.4l6.5-13.7l6.5,13.7l14.8-3.4l-6.6,13.6l11.9,9.4
+	l-14.8,3.3L452.7,230.1z M23.7,0L89,65.3V0H23.7z M0,0v25.2L63.9,89H89L0,0z M0,36.5V89h52.5L0,36.5z"/>
+<polygon class="st1" points="89,0 104,0 104,104 0,104 0,89 52.5,89 0,36.5 0,25.2 63.9,89 89,89 0,0 23.7,0 89,65.3 "/>
+<polygon class="st1" points="104,152 104,256 89,256 89,178.3 11.3,256 0,256 89,167 63.9,167 0,230.8 0,207.1 40.1,167 0,167 
+	0,152 "/>
+<polygon class="st1" points="192.1,89 256,25.2 256,48.9 215.9,89 256,89 256,104 152,104 152,0 167,0 167,77.7 244.7,0 256,0 
+	167,89 "/>
+<polygon class="st1" points="256,167 203.5,167 256,219.5 256,230.8 192.1,167 167,167 256,256 232.3,256 167,190.7 167,256 
+	152,256 152,152 256,152 "/>
+<polygon class="st0" points="167,167 256,256 256,230.8 192.1,167 "/>
+<polygon class="st0" points="89,89 0,0 0,25.2 63.9,89 "/>
+<polygon class="st1" points="128,307.5 141.7,336.1 172.6,329 158.8,357.5 183.7,377.3 152.7,384.2 152.8,416 128,396.1 103.2,416 
+	103.3,384.2 72.3,377.3 97.2,357.5 83.4,329 114.3,336.1 "/>
+<polygon class="st1" points="387.8,344 394.4,357.6 409.1,354.2 402.5,367.9 414.4,377.3 399.6,380.6 399.7,395.7 387.8,386.3 
+	376,395.7 376.1,380.6 361.3,377.3 373.2,367.9 366.6,354.2 381.3,357.6 "/>
+<polygon class="st1" points="327.1,199.1 333.6,212.7 348.4,209.3 341.7,222.9 353.6,232.4 338.8,235.7 338.9,250.8 327.1,241.4 
+	315.3,250.8 315.3,235.7 300.5,232.4 312.4,222.9 305.8,209.3 320.5,212.7 "/>
+<polygon class="st1" points="387.8,116.3 394.4,129.9 409.1,126.5 402.5,140.1 414.4,149.5 399.6,152.9 399.7,168 387.8,158.5 
+	376,168 376.1,152.9 361.3,149.5 373.2,140.1 366.6,126.5 381.3,129.9 "/>
+<polygon class="st1" points="440.9,178.4 447.5,192 462.2,188.6 455.6,202.2 467.5,211.6 452.7,215 452.7,230.1 440.9,220.6 
+	429.1,230.1 429.2,215 414.4,211.6 426.3,202.2 419.6,188.6 434.4,192 "/>
+<polygon class="st1" points="403,250.8 408.1,266.6 424.7,266.6 411.3,276.4 416.4,292.2 403,282.5 389.5,292.2 394.7,276.4 
+	381.2,266.6 397.8,266.6 "/>
+<polygon class="st2" points="0,152 104,152 104,256 152,256 152,152 256,152 256,104 152,104 152,0 104,0 104,104 0,104 "/>
+<g>
+	<polygon class="st2" points="167,167 256,256 256,230.8 192.1,167 	"/>
+	<polygon class="st2" points="63.9,167 0,230.8 0,256 0,256 89,167 	"/>
+</g>
+<g>
+	<polygon class="st2" points="89,89 0,0 0,25.2 63.9,89 	"/>
+	<polygon class="st2" points="192.1,89 256,25.2 256,0 256,0 167,89 	"/>
```

Comment:
I think these coordinates have unnecessary many decimals, plus there's excess whitespace in the last 6 lines.